packages/main/src/Menu.ts

```diff
@@ -253,7 +253,9 @@ class Menu extends UI5Element {
 	loadingDelay!: number;
 
 	/**
-	 * Defines the ID or DOM Reference of the element that the menu is shown at
+	 * Defines the ID or DOM Reference of the element that the menu is shown at.
```

Comment:
I suggest:
Defines the ID or DOM Reference of the element at which the menu is shown.